demo-python/code/vector-quantization-and-storage/README.md

```diff
@@ -0,0 +1,42 @@
+---
+page_type: sample
+languages:
+  - python
+name: Vector quantization and storage options in Python
+products:
+  - azure
+  - azure-ai-search
+description: |
+  Using azure-search-documents and the Azure SDK for Python, save on storage when using vector quantization
+urlFragment: quantization-storage-python
+---
+
+# Vector quantization and storage options using Python (Azure AI Search)  
+
+The Python notebook creates vectorized data on Azure AI Search and demonstrates how to save on storage using the following features
+
+- Quantization
```

Comment:
line 18, if you took the edit for C#, the same edit here is "Built-in scalar quantization that reduces vector index size in memory and on disk."